cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -58,7 +59,7 @@ var fTest = flag.Bool("test", false, "enable test mode: gather metrics, print th
 var fTestWait = flag.Int("test-wait", 0, "wait up to this many seconds for service inputs to complete in test mode")
 var fSchemaTest = flag.Bool("schematest", false, "validate the toml file schema")
 var fConfig = flag.String("config", "", "configuration file to load")
```

Comment:
nit: Unrelated to your change, but now that we have two configuration files that get read in can we rename the variable and description to include that it's for the telegraf portion or TOML configuration?

Reply:
Can do